src/org/labkey/test/tests/WorkbookTest.java

```diff
@@ -89,19 +90,21 @@ public void testSteps()
         assertEquals("id's generated when workbooks are created should be sequential", Arrays.asList(1, 2, 3), ids);
 
         // Edit Workbook Name
-        waitAndClick(Locator.css(".wb-name + .labkey-edit-in-place"));
-        WebElement nameInput = waitForElement(Locator.css(".wb-name + .labkey-edit-in-place + input"));
+        WebElement wbNameEl = Locator.css(".wb-name + .labkey-edit-in-place").waitForElement(shortWait());
```

Comment:
this seems like the sort of thing that would go nicely into a component, if we foresee needing to do that again